Signal/src/UIColor+OWS.m

```diff
@@ -107,4 +107,21 @@ + (UIColor *)backgroundColorForContact:(NSString *)contactIdentifier
     return [colors objectAtIndex:(choose % [colors count])];
 }
 
++ (UIColor *)colorWithRGBHex:(unsigned long)value
+{
+    CGFloat red = ((value >> 16) & 0xff) / 255.f;
+    CGFloat green = ((value >> 8) & 0xff) / 255.f;
+    CGFloat blue = ((value >> 0) & 0xff) / 255.f;
+    return [UIColor colorWithRed:red green:green blue:blue alpha:1.f];
+}
+
++ (UIColor *)colorWithARGBHex:(unsigned long)value
```

Comment:
Huh.  Well, having done some googling I see you're right there is no clearly right answer.  I've just always seen hex colors use ARGB, but I guess that was anecdotal.  I'm going to remove this method since we're not even using it yet.